README.md

```diff
@@ -240,6 +242,31 @@ Or, using Sass:
 </button>
 ```
 
+
+## Troubleshooting
+
+In the past we've seen issues where adding styles to the `background` property of buttons with a low specificity selector can overwrite the `background-color` styles added by o-buttons, [see issue #76](https://github.com/Financial-Times/o-buttons/issues/76). This happens because o-buttons styles the `background-color` property on buttons to add color instead of the shorthand property, which allows for icons to be added to buttons using the remaining background properties.
+
+To avoid this, use the `background-color` property instead of the shorthand `background` property if you wish to overwrite a buttons background-color.
+
+## Migration Guide
+
+### Migrating from v4 to v5
+
+This major includes the new o-colors and updates the themes and sizes of buttons.
+
+**Sizes** have been updated to `default` (`28px` min-height), and `big` (`40px` min-height) to correspond to the new baseline sizing introduced in the new o-typography. `Small` button size has been removed.
+
+The following changes have been made to the **themes**:
+
+- `Standard` is now `Secondary` and the default button style: use `o-buttons` or `o-buttons--secondary` classes
+- `Standout` is now `Primary`: use `o-buttons--primary` class
+- `Uncolored` is now `Mono`: use `o-buttons--mono` class
+
+Inverse and B2C themes have remained the same.
+
+Removes deprecated classnames: `o-buttons__pagination` and `o-buttons__group`.
```

Comment:
We should mention what the correct classnames are as well, perhaps as a git diff codeblock